kubernetes/resource_kubernetes_daemon_set_v1_test.go

```diff
@@ -994,18 +1000,17 @@ func testAccKubernetesDaemonSetV1ConfigWithResourceRequirements(deploymentName,
         container {
           image = "%s"
           name  = "containername"
+          command = ["sleep", "infinity"]
 
           resources {
             limits = {
               cpu          = "0.5"
               memory       = "512Mi"
-              "nvidia/gpu" = "1"
```

Comment:
note: this is causing the pods are unschedulable and thus rollout fails and the test never passes.